oarepo_vocabularies/ui/theme/assets/semantic-ui/js/oarepo_vocabularies_ui/form/components/LocalVocabularySelectField/LocalVocabularySelectField.jsx

```diff
@@ -87,11 +89,17 @@ export const LocalVocabularySelectField = ({
     );
   }
 
-  const serializedOptions = useMemo(
```

Comment:
Can we do the following computation, while preserving const without reassigning & creating another useMemo? I would use ternary operator here to determine options value

Reply:
@mirekys Hi, but I need to make serialization in any case, regardless of whether I am using leaf only or not and then in case I use leaf only I additionally filter out the non leaves.